tests/sqlgateway/mock/mock_client.py

```diff
@@ -0,0 +1,62 @@
+import os
+from typing import List
+
+from flink.sqlgateway.operation import SqlGatewayOperation
+from flink.sqlgateway.session import SqlGatewaySession
+from flink.sqlgateway.config import SqlGatewayConfig
+from tests.sqlgateway.mock.gw_router import GwRouter
+import requests
+import json
+
+
+class MockFlinkSqlGatewayClient:
```

Comment:
I got very confused by this name. I initially thought this is a replacment for `flink.sqlgateway.client.FlinkSqlGatewayClient` but it is actually a Mock of an SqlGateway and we still use the regular client. So maybe the better name would be just `MockFlinkSqlGateway`?

Reply:
that is a mock sqlgateway